webportal/src/app/job/job-view/job-detail-ssh-info-modal.component.ejs

```diff
@@ -0,0 +1,38 @@
+<!-- Modal -->
+<div id="sshInfoModal" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h4 class="modal-title">SSH to <%= containerId %></h4>
+      </div>
+      <div class="modal-body">
+<%
+        const marked = require('marked');
+        const mdContent = `
+Step 1. Open a Bash shell terminal.
+
+Step 2. Download the [private key](${keyPair.privateKeyDirectDownloadLink}):
+\`\`\`
+wget ${keyPair.privateKeyDirectDownloadLink} -O key
+\`\`\`
+
+Step 3. Set correct permissions for the key file:
+\`\`\`
+chmod 600 key
+\`\`\`
+
+Step 4. Run:
+\`\`\`
+ssh -o StrictHostKeyChecking=no -i key -p ${sshPort} root@${sshIp}
+\`\`\`
+        `;
+        const htmlContent = marked(mdContent);
+%>
```

Comment:
you can use `<pre>` or `<code>` tag in html. otherwise, those instructions should be defined in js and passed as a parameter here?

Reply:
Very good suggestion. I used `marked` to simplify the formatting but the ejs file looks ugly indeed. I'll take your second way you proposed.